scalafix-core/src/main/scala/scalafix/v1/Symbol.scala

```diff
@@ -38,6 +38,18 @@ final class Symbol private (val value: String) {
       case _ => false
     })
   override def hashCode(): Int = value.##
+
+  private def encode(value: String): String = {
```

Comment:
This method doesn't handle some cases that `scala.meta.Term.Name` handles, for example keywords
```
@ scala.meta.Term.Name("for").toString
res4: String = "`for`"
```

The motivation to call it "displayName" instead of "name" was to make it emphasize that this name is not a valid `scala.meta.Name`, although I admit this fact is not obvious when I think of it now.